Makefile

```diff
@@ -1,2 +1,5 @@
 host:
 	python3 -m http.server --bind 127.0.0.1 8000
+
+lint: 
+	npx htmlhint -r "tagname-lowercase,attr-lowercase,attr-value-double-quotes,doctype-first,doctype-html5,tag-pair,spec-char-escape,id-unique,src-not-empty,attr-no-duplication,title-require,attr-unsafe-chars,href-abs-or-rel,html-lang-require,id-unique,input-requires-label,tag-pair,tagname-lowercase,tagname-specialchars,attr-no-unnecessary-whitespace,empty-tag-not-self-closed"
```

Comment:
If we make an `.htmlhintrc`, this line can be `npx htmlhint`, and each of these comma-separated items can be on their own line, like [here](https://gist.github.com/oogatta/3654604f9dff9347beb76900a95b6da5). Since git tracks changes line-by-line, this means that changes to the configuration identify the specific lines that were changed.
I look to make it a config file the moment it's more than just a few items long

Reply:
Thank you @wknowleskellett , i'll take care of these when i resubmit